Dockerfile

```
@@ -1,5 +1,5 @@
 ARG FLUENTD_ARCH
-FROM ruby:2.6.9-buster AS builder
+FROM ruby:2.7.5-buster AS builder
```

```suggestion
FROM ruby:2.7.5-bullseye AS builder
```

in order to fix ffi issue